packages/arb-token-bridge-ui/src/components/common/PendingTransactions.tsx

```diff
@@ -0,0 +1,73 @@
+import { motion } from 'framer-motion'
+import { InformationCircleIcon } from '@heroicons/react/outline'
+import Loader from 'react-loader-spinner'
+import { OutgoingMessageState } from 'token-bridge-sdk'
+import { DepositStatus, MergedTransaction } from '../../state/app/state'
+import { outgoungStateToString } from '../../state/app/utils'
+import { motionDivProps } from '../MainContent/MainContent'
+import { DepositCard } from '../TransferPanel/DepositCard'
+import { WithdrawalCard } from '../TransferPanel/WithdrawalCard'
+
+function isDeposit(tx: MergedTransaction) {
```

Comment:
We have the same function in `src/components/TransactionsTable/TransactionsTable.tsx`, can we extract it?